packages/flutter_tools/bin/podhelper.rb

```diff
@@ -190,7 +191,7 @@ def flutter_install_ios_engine_pod(ios_application_path = nil)
   }
 
   # Keep pod path relative so it can be checked into Podfile.lock.
-  pod 'Flutter', :path => 'Flutter'
+  pod 'Flutter', :path => flutter_relative_path_from_podfile(podspec_directory)
```

Comment:
This is now called from the host app Podfile script, and the fake podspec is in the Flutter module, so don't assume they are siblings.